<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>691</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>ME909S-821 check</string>
    </property>
    <widget class="QCheckBox" name="checkBox_deviceNode">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>deviceNode</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_ltemodule">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LTEmodule</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SIMSlot">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMSlot</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SIMDataService">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMDataService</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SIMOperator">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMOperator</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SIMDialing">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMDialing</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SIMSignal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMSignal</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_netAccess">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>NetAccess</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_deviceNode">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_LTEmodule">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_SIMslot">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_service">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_operator">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dialing">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_signal">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_netaccess">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>210</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_temp">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_temp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>temperature</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_version">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
